.travis.yml

```diff
@@ -1,10 +1,14 @@
 sudo: required
+env:
+- PUBLIC_REVERSE_DEPENDENCIES=cli
```

Comment:
Do we need to support multiple dependencies? Each Dockerfile will only have one parent.

Reply:
Yes, but a parent can have multiple reverse dependencies?

Reply:
You mean children? :) We should probably just call them that for clarity?

Reply:
Yup, thought "reverse dependencies" is more general for consistency in case we apply that to other repositories, but I suppose we can call these children for now :)